### PR TITLE
Add `wash build` docs

### LINKED ISSUE #501: Docs: Add `wash build` docs to wasmcloud.dev
Currently we have no documentation in this repo or on wasmcloud.dev for how `wash build` works and the various options for configuring it with `wasmcloud.toml`. I think our best option is to add documentation to wasmcloud.dev with different `wasmcloud.toml` examples and the recommendation that most people use `wash build`

### PR DESCRIPTION
**Pull request by jordan-rash**
## Feature or Problem
Adds documentation for `wash build`

### CODE REVIEW

**Comment by jordan-rash** [comment]
I deciphered the code as well as I could.....it'd be good if @mattwilkinsonn could sanity check it all

File: README.md [hunk 1]
```diff
@@ -91,6 +92,71 @@ nix run github:wasmCloud/wash
 
 `wash` has multiple subcommands, each specializing in one specific area of the wasmCloud development process.
 
+### build
+
+Builds and signs the actor, provider, or interface as defined in a `wasmcloud.toml` file.  Will look for configuration file in directory where command is being run.
+There are three main of a `wasmcloud.toml` file: common config, language config, and type config.
```
Comment (brooksmtownsend) [inline README.md]: nit: three main sections/parts

**Review by brooksmtownsend — DISMISSED** [review]
This is awesome @jordan-rash ! I'm starting to wonder if we need to start breaking up this README, or otherwise putting this content on wasmcloud.com too, but this is great for now. Left a nit, and @mattwilkinsonn can feel free to drop a review too

**Comment by jordan-rash** [comment]
> or otherwise putting this content on wasmcloud.com too

~This is on my list for later today :-)~

Update.....later this week 😅

**Comment by ghost** [comment]
I will drop a review today! Thank you for writing these @jordan-rash!

File: README.md [hunk 2]
```diff
@@ -91,6 +92,71 @@ nix run github:wasmCloud/wash
...
+
+#### Language Config - [tinygo]
+| Setting     | Type   | Default        | Description                   |
+| ----------- | ------ | -------------- | ----------------------------- |
+| tinygo_path | string | `which tinygo` | The path to the tinygo binary |
+
+#### Language Config - [rust]
+| Setting     | Type   | Default       | Description                             |
+| ----------- | ------ | ------------- | --------------------------------------- |
+| cargo_path  | string | `which cargo` | The path to the tinygo binary           |
```
Comment (ghost) [inline README.md]: The path to the cargo binary

**Review by ghost — DISMISSED** [review]
Thanks @jordan-rash !

File: README.md (hunk 2 above)
Comment (jordan-rash) [reply README.md]: 🦀

**Comment by brooksmtownsend** [comment]
@jordan-rash would you mind addressing the two nits? Then I can merge this in 😄

**Review by brooksmtownsend — APPROVED** [review]

**Review by brooksmtownsend — APPROVED** [review]

**Pull request merged**